docs/release-notes/NuGet-6.13.md

```diff
@@ -23,6 +16,104 @@ NuGet distribution vehicles:
 
 <sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
 
-## Not yet released
+## Summary: What's New in 6.13
+
+* Detect if restore used NuGetAudit or not for PackageReference projects - [#13778](https://github.com/NuGet/Home/issues/13778)
+
+* Support for credential providers deployed via .NET tools - [#12567](https://github.com/NuGet/Home/issues/12567)
+
+### Breaking changes
+
+* Support "Supplied by Platform" scenario in restore - [#7344](https://github.com/NuGet/Home/issues/7344)
+
+### Issues fixed in this release
+
+* Add indicator for deprecated and vulnerable packages to Package Details tab header. - [#13974](https://github.com/NuGet/Home/issues/13974)
+
+* Navigation telemetry for 'Clear All NuGet Storage' Command in VS Settings - [#13968](https://github.com/NuGet/Home/issues/13968)
+
+* New dependency resolver downloads more packages than before - [#13943](https://github.com/NuGet/Home/issues/13943)
+
+* Nuget Packages Manager screen for Solution automatically selects transitive dependencies - [#13893](https://github.com/NuGet/Home/issues/13893)
+
+* Debugging large repos with static graph-based restore is slow - [#13876](https://github.com/NuGet/Home/issues/13876)
+
+* NuGet Client SDK packages updating to net8.0 - [#13842](https://github.com/NuGet/Home/issues/13842)
+
+* Promote NU3043 warning to error in .NET 10 - [#13814](https://github.com/NuGet/Home/issues/13814)
+
+* dotnet nuget why should check RID specific packages - [#13718](https://github.com/NuGet/Home/issues/13718)
```

Comment:
I want to nominate this for elevating it to the what's new section.

It's sparse right now, so this is important.